test/expected-output-test/expressions/print.expected

```diff
@@ -0,0 +1,20 @@
+a(k int).
+
+b(k int,
+  p text,
+  q text,
+  r int).
+
+c(s text,
+  n int,
+  t text).
+
+Q( "test"  :: TEXT, 123, id, unnest(y)) * :-
+    a(id),
+    b(id, x, y, z),
+    c(x || y, 10,  "foo" ), [z > 100]; [[![z < 20]], [z < 50]].
```

Comment:
It seems the condition should be wrapped with an extra `[ ]` when printed?

Also to verify compound expressions are parsed correctly, you can print every expression surrounded by a parentheses.  (Right, it's not pretty but makes things super clear!)